### DISCUSSION TITLE
motioneye dove lo installo?

### DISCUSSION THREAD

**Discussion by issuluca** [indomus/forum#962] [HUB: Home Assistant]
Qualche tempo fa lo avevo installato, ma non mi aveva soddisfatto.
Vorrei riprovarci, dove mi consigliate di installarlo?
ho queste opzioni disponibili:
- come add-on su homeassistant su macchina virtuale proxmox
- su macchina virtuale dedicata, su proxmox
- su portainer
ho 5 cam quindi mi serve il sistema più efficente possibile

**Comment by Nik71git** [comment]
Ciao, io uso 2 imou collegate via wifi con risoluzione 1920x1080 25 frps e 4 iegeek collegate via ethernet 1gbps sempre risoluzione 1920x1080 ma 12 frps. Entrambi i settaggi sono lo stream 1 nativo delle cam e già quiesto aiuta molto motioneyeos (e anche motioneye addon...) a non sudare perchè non fa transcodifica. Il lag è sempre attorno ai 6 secondi (sia in Motioneyeos che via onvif su HA), premetto che motioneye non fa streaming verso HA, Ha si arrangia da solo appunto con onvif.
Il lag però è solo "visivo" in quanto il rilevamento motion (di cui ricevo notifica via webhook) è immediato (1 secondo), cioè mi metto di fronte ad una cam con lo smartphone in mano, alzo un braccio, la notifica su HA è immediata ma vedo il movimento sullo schermo dello smartphone da app di HA dopo circa 6 secondi. Se invece uso la app delle cam (quelle iegeek le vede in local mentre dahua solo cloud) il movimento è in tempo reale... Questo rimane per me ancora un mistero tecnico... ma al momento mi va bene così.
L'hardware sono due Raspberry Pi4 4 Gb Ram, uno dedicato ad Hassio ed uno dedicato a MotioneyeOS. Per quel che mi riguarda ho trovato una soluzione che mi soddisfa in quanto le notifiche motion possono essere usate come trigger per eventuali automazioni. Mi sono spiegato bbbenene ??
